projects/m2/016-reverse-lookup/js/index.js

```diff
@@ -0,0 +1,28 @@
+// Create a dictionary
+const dictionary = {
+    a: 1,
+    b: 2,
+    c: 3,
+    d: 4,
+    e: 1,
+}
+
+const userValue = parseInt(prompt('Enter a number', '1'))
+//Function reverseLookup finds all of the keys in a dictionary that map to a specific value
+function reverseLockup(obj, value){
+    //Create empathy array for saving the keys
+    const keyObject = []
+
+    //Check if userValue exists in the dictionary
+    for(let item in obj){
+        if(obj[item] === value){
```

Comment:
Try to solve the exercise by **considering also** the following documentation:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries